src/api.rs

```diff
@@ -0,0 +1,50 @@
+use clap::Parser;
+use clap_verbosity_flag::{InfoLevel, Verbosity};
+
+#[derive(Debug, Parser)]
+#[clap(author, version, about)]
+pub struct Cli {
+	#[clap(
+		long,
+		short,
+		env = "CLOUDFLARE_RECORDS",
+		value_name = "RECORDS",
+		value_delimiter(',')
+	)]
+	/// Comma separated DNS records to update with the host's public IP
```

Comment:
this might be a nit, but it's throwing me off to have the comment sandwiched between the macro and variable, rather than being on top of the macro. At first I thought the comments were trailing their code definitions.

Reply:
Actually I agree, probably be better to start with the comment honestly. Wasn't paying much attention I guess.